src/core/job.c

```diff
@@ -693,10 +693,12 @@ _pure_ static const char *job_get_status_message_format(Unit *u, JobType t, JobR
         return NULL;
 }
 
+#define ANSI_OK_COLOR "ANSI_" OK_COLOR
```

Comment:
hmm, wut? not following here... this is supposed to map to a string containing an ANSI sequence, but not the literal string "ANSI_", no? What am I missing here?

Reply:
This was a mistake.